src/component/Invoice/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react"
+import moment from "moment"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons"
+
+function Invoice({ invoiceTotal, invoiceStatus, invoiceStatusName, clientName, invoiceId, dueDate }) {
+
+    const date = moment(dueDate);
+    const formattedDate = date.format("DD MMM YYYY");
+    const element = <FontAwesomeIcon icon={faChevronRight} />
+
+
+    useEffect(setColor, [])
+
+    const [statusColor, setStatusColor] = useState('')
+    const [statusBorderColor, setStatusBorderColor] = useState('')
+
+    function setColor() {
```

Comment:
Suggest renaming this function - it could cause confusion becuase it starts with set just like our state setter functions (like setStatusColor etc)